source/frontend/src/components/common/GenericModal.tsx

```diff
@@ -211,19 +208,17 @@ export const GenericModal = (props: Omit<BsModalProps, 'onHide'> & ModalProps) =
       onHide={noop}
       className={getModalClass()}
     >
-      <Modal.Header closeButton={closeButton} onHide={close}>
+      <Modal.Header onHide={close}>
```

Comment:
This closebutton actually causes a few problems.

Clicking this close button does not fire our close event.
the styling is hard to see and does not use one of our approved icons.